libraries/AP_HAL_ChibiOS/hwdef/CubeRedPrimary/hwdef.dat

```diff
@@ -105,15 +105,15 @@ PE5    SPI4_MISO    SPI4
 PE6    SPI4_MOSI    SPI4
 
 # Sensors
-SPIDEV icm42688_0       SPI1 DEVID1  ICM42688_0_CS      MODE3  2*MHZ  8*MHZ
+SPIDEV icm42688_0       SPI1 DEVID1  ICM42688_0_CS      MODE3 20*MHZ 20*MHZ
 SPIDEV ms5611_0         SPI1 DEVID2  BARO_0_CS          MODE3 20*MHZ 20*MHZ
 
-SPIDEV icm42688_1       SPI2 DEVID1  ICM42688_1_CS      MODE3  2*MHZ  8*MHZ
+SPIDEV icm42688_1       SPI2 DEVID1  ICM42688_1_CS      MODE3  8*MHZ  8*MHZ
 SPIDEV rm3100           SPI2 DEVID2  RM3100_CS          MODE3  1*MHZ  1*MHZ
 
 SPIDEV icm20649         SPI4 DEVID1  ICM_2_CS           MODE3  4*MHZ  8*MHZ
 # alternative to icm20649
-SPIDEV icm45686         SPI4 DEVID1  ICM_2_CS           MODE3  2*MHZ  8*MHZ
+SPIDEV icm45686         SPI4 DEVID1  ICM_2_CS           MODE3 20*MHZ 20*MHZ
```

Comment:
I thought the slower rate was slower for a reason, i.e. in case of recovery?

Reply:
@andyp1per the reason is simply legacy, we have been using that frequency in older Sensors.

Reply:
Yeah, but for instance we needed that to solve a problem on ICM42688 (changed it on Matek for instance), so not 100% convinced this is an older sensor problem.